lib/samples/apply_scheduled_updates_to_preplanned_map_area/apply_scheduled_updates_to_preplanned_map_area_sample.dart

```diff
@@ -30,80 +31,163 @@ class ApplyScheduledUpdatesToPreplannedMapAreaSample extends StatefulWidget {
+  // The Active mobile map package
   MobileMapPackage? _mobileMapPackage;
+  // Offline task and parameters used for updating the map package
+  OfflineMapSyncTask? _offlineMapSyncTask;
   OfflineMapSyncParameters? _mapSyncParameters;
+  // The location of the map package on the device
   late final Uri _dataUri;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.center,
+        top: false,
+        child: Stack(
           children: [
-            Expanded(
-              child: ArcGISMapView(
-                controllerProvider: () => _mapViewController,
-                onMapViewReady: onMapViewReady,
-              ),
-            ),
-            SizedBox(
-              height: 75,
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                crossAxisAlignment: CrossAxisAlignment.center,
-                children: [
-                  Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                    children: [
-                      Text('Updates: ${_updateStatus.name.toUpperCase()}'),
-                      Text('Update Size: ${_updateSizeKB}KB'),
-                    ],
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Expanded(
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
                   ),
-                  Center(
-                    child: ElevatedButton(
-                      onPressed: _canUpdate ? syncUpdates : null,
-                      child: const Text('Update'),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        Text('Updates: ${_updateStatus.name.toUpperCase()}'),
+                        Text('Update Size: ${_updateSizeKB}KB'),
+                      ],
                     ),
-                  ),
-                ],
+                    Center(
+                      child: ElevatedButton(
+                        onPressed: _updating
+                            ? null
+                            : _canUpdate
+                                ? syncUpdates
+                                : resetMapPackage,
+                        child: _canUpdate
+                            ? const Text('Update')
+                            : const Text('Reset'),
```

Comment:
The reset workflow isn't working for me - in that the button stays as "Reset" even after I've tapped it. However that said, we don't actually need a reset button per se, only an "Apply updates" button. The purpose of the sample workflow is to show how you can check for any updates when the app opens and choose to apply the updates if they're available.

What we do need to incorporate, is that the next time you re-open the sample it allows you to apply the updates again (whereas before you had to delete the app). I think the zip archive approach you've implemented will still work - you'll just check at the start whether to re-download or just re-extract. But then you can simplify the sample by removing the user controlled / UI based reset workflow. I hope I'm making sense!!

Reply:
The reset logic has been removed. The map will reset when the sample is restarted.